tests/Zumba/Util/Test/JsonSerializerTest.php

```diff
@@ -226,6 +226,20 @@ public function testSerializationOfDateTime() {
 	}
 
 	/**
+	 * Test serialization of DateTime classes
+	 *
+	 * Some interal classes, such as DateTime, cannot be initialized with
+	 * ReflectionClass::newInstanceWithoutConstructor()
+	 *
+	 * @return void
+	 */
+	public function testSerializationOfDateTime() {
```

Comment:
@wrey75 This method is already defined. You have to rename it. The docblock seems copied from the other method.